Lib/unittest/test/testmock/testthreadingmock.py

```diff
@@ -0,0 +1,173 @@
+import threading
+import multiprocessing
+import time
+import unittest
+import concurrent.futures
+
+from test.support import start_threads
```

Comment:
Doesn't seem to be used.

Reply:
Good catch, I refactored this to work with futures, I need neither that nor multiprocessing nor threading.